ActionSheetPicker-3.0.podspec

```diff
@@ -15,7 +15,6 @@ Pod::Spec.new do | s |
                         "http://skywinder.github.io/ActionSheetPicker-3.0/Screenshots/string.png"]
     s.requires_arc = true
     s.ios.deployment_target = '6.1'
-    s.platform = :ios
```

Comment:
probably it's better to specify version rather than delete platfor specification?

Reply:
probably it's better to specify version rather than delete platform specification?